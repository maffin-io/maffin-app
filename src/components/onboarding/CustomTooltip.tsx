import React from 'react';
import { TooltipRenderProps } from 'react-joyride';

export default function CustomTooltip({
  step,
  tooltipProps,
}: TooltipRenderProps): JSX.Element {
  return (
    <div
      className="card m-0 mx-auto w-1/2 text-sm rounded-md"
      {...tooltipProps}
    >
      {step.content}
    </div>
  );
}
