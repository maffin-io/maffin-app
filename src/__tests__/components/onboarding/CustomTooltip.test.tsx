import React from 'react';
import { render, screen } from '@testing-library/react';

import CustomTooltip from '@/components/onboarding/CustomTooltip';

describe('CustomTooltip', () => {
  it('renders as expected', () => {
    const { container } = render(
      <CustomTooltip
        step={{
          content: 'Hello',
        }}
      />,
    );

    screen.getByText('Hello');
    expect(container).toMatchSnapshot();
  });
});
