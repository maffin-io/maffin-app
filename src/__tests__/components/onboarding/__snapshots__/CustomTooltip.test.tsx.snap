// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTooltip renders as expected 1`] = `
<div>
  <div
    class="card m-0 mx-auto w-1/2 text-sm rounded-md"
  >
    Hello
  </div>
</div>
`;
